rnaget-openapi.yaml

```
@@ -1406,7 +1406,7 @@ components:
     featureMinParam:
       name: feature_min_value
       in: query
-      description: Sets a minimum expression value for features.  If set the resulting matrix will not contain any features with a value less than or equal to the provided threshold in any sample in the matrix.  The resulting matrix should have values > the supplied threshold in every cell.  For example, specifying a feature_min_value of 0 will remove zeros from the matrix by removing any feature that has a zero value in 1 or more samples.  Using this with feature_max_value will result in matrix values in the half open interval (feature_min_value, feature_max_value].
+      description: Sets a minimum expression value for features.  If set the resulting matrix will not contain any features with a value less than the provided threshold in any sample in the matrix.  The resulting matrix should have values >= the supplied threshold in every cell.  Using this with feature_max_value will result in matrix values in the half closed interval [feature_min_value, feature_max_value].
```

the word `half` should be removed
```suggestion
      description: Sets a minimum expression value for features.  If set the resulting matrix will not contain any features with a value less than the provided threshold in any sample in the matrix.  The resulting matrix should have values >= the supplied threshold in every cell.  Using this with feature_max_value will result in matrix values in the closed interval [feature_min_value, feature_max_value].
```